gstd/gstd_signal_reader.c

```diff
@@ -197,6 +197,7 @@ gstd_signal_reader_read_signal (GstdIReader * iface,
   self->callback = NULL;
 
 out:
+  g_object_unref (target);
   g_signal_handler_disconnect (target, handler_id);
```

Comment:
`target` in `g_signal_handler_disconnect` is an unref pointer, did you not get an issue?

Reply:
g_object_get() generates reference that has to be decreased in order to clean target (see [this](https://developer.gnome.org/gobject/stable/gobject-The-Base-Object-Type.html#g-object-get) documentation.)